src/js/chapter-data/englishData.js

```diff
@@ -0,0 +1,218 @@
+const englishData = [{
+  // causes it to not show up on the homepage
```

Comment:
is that comment correct or just left over?

Reply:
That's probably left over from demoData.js (from the original chapter 999).

Reply:
mind removing it? thanks